packages/share/ui/src/components/navigation-menu.tsx

```diff
@@ -83,7 +83,7 @@ const NavigationMenuViewport = React.forwardRef<
   React.ElementRef<typeof NavigationMenuPrimitive.Viewport>,
   React.ComponentPropsWithoutRef<typeof NavigationMenuPrimitive.Viewport>
 >(({ className, ...props }, ref) => (
-  <div className={cn('absolute left-0 top-full flex justify-center')}>
+  <div className={cn('absolute -right-14 top-full flex justify-center')}>
```

Comment:
The component takes `className` as props. We should pass className instead of changing its default style.

Reply:
but this one is nested in inside that tailwind class cannot reach

Reply:
So is there any solution on this

Reply:
We can revert the change now.
The submenu is moved to the "sketch page" as tabs.